packages/dd-trace/src/opentelemetry/otlp/otlp_transformer_base.js

```diff
@@ -61,7 +45,7 @@ class OtlpTransformerBase {
 
     for (const item of items) {
       const instrumentationScope = item.instrumentationScope || { name: '', version: '', schemaUrl: '', attributes: {} }
-      const attrsKey = stableStringify(instrumentationScope.attributes || {})
+      const attrsKey = JSON.stringify(instrumentationScope.attributes || {})
```

Comment:
If attributes are an object we must use a stable stringify implementation. Otherwise the keys would be stringified in insertion order and it would not match anymore.

Reply:
Also added a test case to catch this